app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -247,5 +274,35 @@ Object getItem(int position) {
         return items.get(position);
     }
 
+
+    private int getItemCount(boolean excludeFooter) {
+        if (excludeFooter) return getItemCount();
+        else return getItemCount() - 1;
+    }
+
+    void isFooterShowing(boolean show) {
```

Comment:
`is__` style functions typically return a boolean (ask a question).  A better name for this function might be `setFooterVisibility`.